src/app/core/auth.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  baseUrl = "https://asidjaskldhaksudhauskd.herokuapp.com/";
```

Comment:
we won't be hard coding base url

Reply:
We should get this URL from environment variables!